src/multinomials.jl

```diff
@@ -0,0 +1,54 @@
+# Multinomial theorem
+# https://en.wikipedia.org/wiki/Multinomial_theorem
+
+export multiexponents
+
+immutable MultiExponents{T}
+    c::Combinations{T}
+    nterms::Int
+end
+
+start(m::MultiExponents) = start(m.c)
+
+# standard stars and bars
```

Comment:
What does "stars and bars" refer to in the context?  Please add a URL or reference.